shared/common-adapters/index.js

```diff
@@ -11,14 +11,22 @@ const MaybePopup = isMobile
     )
   : (props: {
       onClose: () => void,
+      onMouseUp?: (e: SyntheticMouseEvent<>) => void,
+      onMouseDown?: (e: SyntheticMouseEvent<>) => void,
+      onMouseMove?: (e: SyntheticMouseEvent<>) => void,
       children: React.Node,
       cover?: boolean,
       styleCover?: any,
       styleContainer?: any,
     }) => (
       <PopupDialog
         onClose={props.onClose}
-        styleCover={props.cover ? {..._styleCover, ...props.styleCover} : {}}
+        onMouseUp={props.onMouseUp}
+        onMouseDown={props.onMouseDown}
+        onMouseMove={props.onMouseMove}
+        styleCover={
+          props.cover ? {..._styleCover, ...props.styleCover} : props.styleCover ? props.styleCover : {}
```

Comment:
this should just use collapse styles, its smart
```
collapseStyles([props.cover && _styleCover, props.styleCover]}
```

Reply:
Oh, nice! Will do.